_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -32,10 +32,10 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML element][] that has [visible][] [children][] in the [flat tree][] for which at least one of the following is true:
+This rule applies to any non-embedded [scrollable element][] that has [visible][] [children][] in the [flat tree][] for which at least one of the following is true:
```

Comment:
We can also update the list by saying "the element has…" rather than "it has…" which would make it more obvious, and also would better comply with [our own guidelines](https://act-rules.github.io/pages/design/rule-design/#listed-conditions).